README.md

```diff
@@ -53,6 +61,7 @@ client
 | --- | --- | --- | --- |
 | retries | `Number` | 3 | The number of times to retry before failing |
 | retryCondition | `Function` | `isNetworkOrIdempotentRequestError` | A callback to further control if a request should be retried.  By default, it retries if it is a network error or a 5xx error on an idempotent request (GET, HEAD, OPTIONS, PUT or DELETE). |
+| delayStrategy | `Function` | `noDelay` | A callback to further control the delay between retried requests. By default there is no delay between retries. Another option is exponentialDelay ([Exponential Backoff](https://developers.google.com/analytics/devguides/reporting/core/v3/errors#backoff)). |
```

Comment:
I'd use `retryDelay` as it's more consistent with the other options.
Also, the default value is `0` instead of `noDelay`.